funcframework/framework.go

```diff
@@ -86,72 +82,98 @@ func RegisterEventFunction(path string, fn interface{}) {
 
 // RegisterHTTPFunctionContext registers fn as an HTTP function.
 func RegisterHTTPFunctionContext(ctx context.Context, path string, fn func(http.ResponseWriter, *http.Request)) error {
-	server, err := wrapHTTPFunction(path, fn)
-	if err == nil {
-		handler = server
+	if err := registry.Default().RegisterHTTP(path, fn, registry.WithPath(path)); err != nil {
```

Comment:
Oh I kind of forgot these didn't have names... Maybe we should give it a readable name like `fmt.Sprintf("function at path %q", path)` or something so that error messages that print the function name won't look as weird.

Reply:
Done.